node/src/crypto/asymmetric_key_ext.rs

```diff
@@ -139,7 +139,7 @@ impl AsymmetricKeyExt for SecretKey {
                 der = Der::new(&mut encoded);
                 der.sequence(|der| {
                     der.integer(&[1])?;
-                    der.octet_string(secret_key)?;
+                    der.octet_string(secret_key.to_bytes().as_slice())?;
```

Comment:
`to_bytes` is definitely not what we want here :(

We need to do `.into::<Vec<u8>>()`